src/js/player.js

```diff
@@ -1536,6 +1572,33 @@ class Player extends Component {
     }
   }
 
+  /**
+   * Handle a double-click on the media element to enter/exit fullscreen
+   *
+   * @param
+   *
+   * @private
+   */
+  handleTechDoubleClick_(event) {
+    if (!this.controls_) {
+      return;
+    }
+
+    const inAllowedEls = Array.prototype.some.call(
+        this.$$('.vjs-control-bar, .vjs-modal-dialog'),
+        function(el) {
+          el.contains(event.target);
```

Comment:
you want to return from this method or change the `function` to an arrow function.